app/models/product.rb

```diff
@@ -24,7 +25,7 @@ class Product < ApplicationRecord
   has_many :metrics, as: :ownable, dependent: :destroy
 
   validates :name, presence: true, uniqueness: true
-
+  validates :enabled, presence: true
```

Comment:
Would it be a good idea to set `enabled` at the default_scope? Just asking. https://apidock.com/rails/ActiveRecord/Base/default_scope/class

Reply:
I didn't know about default_scope, it's a good idea.